00_Base/json-schema-processor.js

```diff
@@ -35,7 +52,11 @@ fs.readdir(path, (error, files) => {
 
             if (enums.length > 0) {
                 enums.forEach(entry => {
-                    const { enumName, enumDocumentation, enumDefinition } = splitEnum(entry);
+                    let { enumName, enumDocumentation, enumDefinition } = splitEnum(entry);
+                    if (enumName == 'DataEnumType') { // Adding missing type for DataEnumType... type in OCPP 2.0.1 appendix but not in part 3 JSON schemas
```

Comment:
my IDE doesn't have a warning, but I'm guessing that yours is complaining because enumName has 'any' typing. Comparing 'any' to 'string' is safe at runtime, and this code works (it was used to generate the schemas/types in this branch)